src/dispatch/incident/scheduled.py

```diff
@@ -319,6 +319,10 @@ def incident_report_weekly(db_session: Session, project: Project):
         except Exception as e:
             log.exception(e)
 
+    # it's possible all incidents were restricted, so exit instead of sending an empty report
+    if not items_grouped:
+        return
```

Comment:
Should we still send a report, but say there are no incidents to report? If we don't send it, then one could wonder if something went wrong when we tried to generate it. At the very least, perhaps log an informational message that we didn't have incidents to report.

Reply:
I like the idea of sending the notification indicating there were no incidents to report.